app/domain/logs.rb

```diff
@@ -188,6 +188,20 @@ module AuthnAzure
       )
 
     end
+
+    module AuthnGCP
+
+      ExtractingRestrictionsFromResource = ::Util::TrackableLogMessageClass.new(
+          msg: "Extracting restrictions from resource '{0-resource-id}' with prefix '{1-prefix}'",
+          code: "CONJ00039D"
+      )
+
+      ExtractedResourceRestrictions = ::Util::TrackableLogMessageClass.new(
+          msg: "{0-amount} restrictions were extracted",
```

Comment:
@shulifink please review

Reply:
i would write `"{0-amount} Redsource Restrictions were extracted"`

Reply:
Did you meant without the upper case?
`{0-amount} resource restrictions were extracted`

Reply:
*with* the upper case

Reply:
why with upper case?